r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -719,6 +719,55 @@ test_that("errors in strptime", {
   )
 })
 
+test_that("strftime", {
+  # TODO: consider reevaluating this workaround after ARROW-12980
+  withr::local_timezone("UTC")
+  t_stamp <- tibble(x = c(lubridate::ymd_hms("2018-10-07 19:04:05"), NA))
+  t_string <- tibble(x = c("2018-10-07 19:04:05.000000", NA))
+  t_string_2 <- tibble(x = c("07 October 2018 19:04:05.000000 UTC", NA))
+
+  # TODO: remove once we have tz support on windows ARROW-13168
+  if (tolower(Sys.info()[["sysname"]]) != "windows") {
+    expect_equal(
+      t_stamp %>%
+        Table$create() %>%
+        mutate(
+          x = strftime(x)
+        ) %>%
+        collect(),
+      t_string
+    )
+
+    expect_equal(
+      t_stamp %>%
+        Table$create() %>%
+        mutate(
+          x = strftime(x, format = "%Y-%m-%d %H:%M:%S")
+        ) %>%
+        collect(),
+      t_string
+    )
+
+    expect_equal(
+      t_stamp %>%
+        Table$create() %>%
+        mutate(
+          x = strftime(x, format = "%d %B %Y %H:%M:%S %Z", locale = "C")
+        ) %>%
+        collect(),
+      t_string_2
+    )
+  } else {
+    expect_error(
+      t_stamp %>%
+        Table$create() %>%
+        mutate(x = strftime(x)) %>%
+        collect(),
+      "Strftime not yet implemented on windows."
+    )
+  }
```

Comment:
Please can you refactor this block to, instead of using a conditional, use the `skip()` function.  There should be some examples in other test files you can follow, but let me know if it's not clear :)

Reply:
Switched to `skip_on_os("windows")`.